apisix/plugins/openid-connect.lua

```diff
@@ -86,7 +86,7 @@ local schema = {
         },
         redirect_uri = {
             type = "string",
-            description = "use ngx.var.request_uri if not configured"
+            description = "auto append '.apisix/redirect' if not configured"
```

Comment:
Could you clarify what you will append `'.apisix/redirect` to ?

Reply:
> Well. If the route is of type **exact**, then it cannot actually use this `openid-connect` plugin, because the `redirect_uri` must be different from the request URI, as required by `lua-resty-openidc`.

I think this explanation is correct